sentiment_with_wordcloud.Rmd

```diff
@@ -0,0 +1,89 @@
+---
+title : Visualizing Movie Reviews in Word Cloud
+output: html_document
+author: Samrat Halder and Hariz Johnson
+---
+
+```{r , include=FALSE}
+knitr::opts_chunk$set(echo = TRUE)
+```
+# 
```

Comment:
The first line starts with a single hashtag (#) followed by white space and a project name.

Reply:
Second line should be blank.

Reply:
Resolved it. Please check.

Reply:
The first line means that the first line in the whole file. After you remove all the YAML header. The first line in your file should be a single hashtag followed by white space and a project name. The second line should be blank.